lmbdtest/test_delete_tile_objs_lambda.py

```diff
@@ -55,20 +57,35 @@ def test_delete_raises_on_bad_bucket_name(self):
         with self.assertRaises(botocore.exceptions.ClientError):
             handler(event, context)
 
-    @patch('lambdafcns.delete_tile_objs_lambda.delete_keys')
-    def test_delete_raises_on_error_in_resp(self, fake_delete):
-        keys = ['tile1', 'tile2']
+    @moto.mock_s3
+    def test_objects_marked_for_deletion(self):
+        keys = ['tilex', 'tiley']
         event = {
             'region': 'us-east-1',
             'bucket': 'test.bucket.boss',
             'tile_key_list': keys
         }
+
+        s3 = boto3.resource('s3', region_name=event['region'])
+        s3.create_bucket(Bucket=event['bucket'])
```

Comment:
Is this bucket being created in dev account or prod account? I assume dev account right? So if we ever want to run this test for production, we would not be able to right?

Reply:
Also wondering, is this the first test we have where we create a test bucket?

Reply:
This uses moto's mock S3 object. Nothing is created in AWS.

Reply:
My bad.